porch/internal/api/porchinternal/v1alpha1/internal.porch.kpt.dev_pkgrevmetas.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: pkgrevmetas.internal.porch.kpt.dev
+spec:
+  group: internal.porch.kpt.dev
+  names:
+    kind: PkgRevMeta
```

Comment:
I think we can remove `Meta` from the name. `PkgRev` is shorter and can be used to store any info

Reply:
Yeah, I've changed the name of the CRD to `PackageRev`. We want to avoid using the same name as the type for the aggregated apiserver. I've also moved it to the `config.porch.kpt.dev` group. The idea is that we over time want to use this CRD as a replacement for the `PackageRevision` type from the aggregated apiserver, and therefore avoid the internal group and the references to meta.